pkg/services/accesscontrol/actest/common.go

```diff
@@ -59,3 +70,64 @@ func ConcurrentBatch(workers, count, size int, eachFn func(start, end int) error
 	}
 	return nil
 }
+
+// creates a role, connected it to user and store all permission from the user in database
+func AddUserPermissionToDB(t testing.TB, db db.DB, user *user.SignedInUser) {
```

Comment:
this would have been handy when updating some legacy tests, I might need to revisit those

Reply:
All credit goes to Kalle for this one :)